system/firewalld.py

```diff
@@ -162,6 +168,29 @@ def remove_source(zone, source):
     fw_zone.update(fw_settings)
 
 ####################
+# interface handling
+#
+def get_interface(zone, interface):
+    fw_zone = fw.config().getZoneByName(zone)
+    fw_settings = fw_zone.getSettings()
+    if interface in fw_settings.getInterfaces():
+       return True
```

Comment:
Wow, I somehow completely missed the comment thread here. Apologies.

I'm fine either way, there's a major module refactor effort happening elsewhere anyways so I'm +1 for keeping this pattern for now and the refactor can update everything together for consistency.